### DISCUSSION TITLE
AnuPpuccin Discussions

### DISCUSSION THREAD

**Discussion by AnubisNekhet** [AnubisNekhet/AnuPpuccin#17] [Announcements]
## Hey there!

This is the discussions page of AnuPpuccin, where you can share and discuss possible ideas and share your own tweaks and edits with everyone else 😊
I hope this place makes customization and CSS in general more accessible to people!

**Comment by TonkatsuRa** [comment]
How do I use the extended color schemes?
I downloaded the code, put it in a CSS file in the snippet folder and activated it in the settings of obsidian but nothing happened?
What did I do wrong?

**Reply by h259bws** [reply]
Any help on this, please?

**Reply by trainbuffer** [reply]
The Style Settings plugin also needs to be installed. Could that be it?

**Reply by h259bws** [reply]
No, I have Style Settings installed, but it only lists the "built-in" color schemes.

**Reply by AFulgens** [reply]
I've solved it by copy-pasting the whole content of the `extended-colorschemes.css` file at the very end of the `.obsidian\themes\AnuPpuccin\theme.css`.

**Reply by Practical-Seaweed** [reply]
I was able to do this and it worked, so now I can see "AnuPpuccin Themes Extended" in the Style settings but when changing the dark theme flavor nothing happens :( any fixes?

**Comment by ondigitalwings** [comment]
Also having this issue :(

**Reply by SongFuchang** [reply]
I like this ~!!!

**Comment by mowser03** [comment]
Thank you brother , you are a life saver . This was the perfect thing I was looking for  . 🙏

**Comment by CHERRY-YIN** [comment]
我也是下载了这个主题 复制了这个css配色 但是很遗憾 那些颜色和特别的彩虹标我都没有 我有的时候有在想是因为我是win 和Mac的设置路径不同吗
